src/pages/ja/concepts/islands.mdx

```diff
@@ -46,7 +41,7 @@ import MyReactComponent from '../components/MyReactComponent.jsx';
 <MyReactComponent />
 ```
 
-しかし、インタラクティブなUIを作成するためには、クライアントサイドのJavaScriptが必要になります。ページ全体をSPAのようなJavaScriptアプリケーションにするのではなく、Astroはアイランドを作成するように依頼します。
+しかし、インタラクティブなUIを作成するためには、クライアントサイドのJavaScriptが必要になります。Astroでは、ページ全体をSPAのようなJavaScriptアプリケーションにするのではなく、アイランドを作成します。
```

Comment:
原文は

> But sometimes, client-side JavaScript is required for creating interactive UI. Instead of forcing your entire page to become an SPA-like JavaScript application, Astro asks you to create an island.

となります。訳文の最後が「Astroはアイランドを作成するように依頼します」となっているのですが、ここの意味が少し通りづらい（「誰に依頼する？」と読める）気がしたため、読みやすくなるよう変更してみました。